schemas/user.py

```
@@ -0,0 +1,103 @@
+    nickname: str
+    profile_image: str | None
+
+class UserUpdateResponse(UserBase):
+    email: EmailStr
+    nickname: str
+    profile_image: str | None = None
+    created_at: datetime
+
+class TokenData(BaseModel):
+    access_token: str
+    token_type : str = "bearer"
```

Ruff (0.14.11)

Possible hardcoded password assigned to: "token_type"

(S105)